test/unit/features/directives/model-text.spec.js

```diff
@@ -19,6 +19,17 @@ describe('Directive v-model text', () => {
     }).then(done)
   })
 
+  it('should not create a new property ended with spaces when', () => {
```

Comment:
Could you add a test that fails without the fix above please? This one passes
Also, could you rename the test title to `should work with extra whitespace properties` or something similar, please?

Reply:
I did not understand the part of the test. Do I create another pull request with only the test? Because the current test fails without this fix